segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/metadata/BaseMetadataStore.java

```diff
@@ -188,6 +189,7 @@
     /**
      * Keep count of records in buffer. ConcurrentHashMap.size() is an expensive operation.
      */
+    @Getter
```

Comment:
I know this is used just for testing, but you are essentially exposing your inner components to the outside. Someone could take this and mess around with the values inside.

I suggest you convert this to a **packgage-private** hand-written getter and add the `@VisibleForTesting` tag. That way, if you just return the value, nobody could modify the contents.

Reply:
ok

Reply:
good catch. Fixed